packages/ui/src/lib/layerControl/LayerContolGroup.stories.svelte

```diff
@@ -10,25 +10,28 @@
 
 <script lang="ts">
 	let optionsForGroup = [
-		{ id: 'bus', label: 'Bus stops' },
+		{ id: 'bus', name: 'transport', label: 'Bus stops' },
 		{
 			id: 'train',
+			name: 'transport',
 			label: 'Train stations',
 			hint: 'Excluding underground stations'
 		},
-		{ id: 'underground', label: 'Underground stations' },
-		{ id: 'taxi', label: 'Taxi ranks', disabled: true }
+		{ id: 'underground', name: 'transport', label: 'Underground stations' },
+		{ id: 'taxi', name: 'transport', label: 'Taxi ranks', disabled: true }
 	];
```

Comment:
If this is for mutually exclusive layers, the `name` should be set from the `name` prop of the `LayerControlGroup` rather than needing to be separately set for each option. If they're not mutually exclusive, then the `name` for the controls for different layers shouldn't be set to the same value. In either case I'd remove the `name` proprerties from here.